allowedModules.js

```diff
@@ -13,7 +13,8 @@ module.exports = {
     ...sharedWhiteList,
     'criteo-direct-rsa-validate',
     'jsencrypt',
-    'crypto-js'
+    'crypto-js',
+    'live-connect'
```

Comment:
would be nice to have a comment reference here to what this is - maintainer and link? Btw would be nice for all modules here :)